testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1788,63 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', ()=> {
+    QUnit.test('selectByClick option should be changed during initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: false }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
+
+    QUnit.test('selectByClick option should be changed after initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2] }).dxList('instance');
+
+        list.option('selectByClick', false);
+
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
+
+    QUnit.test('onSelectionChanged event shouldn`t call on item click if selectByClick is false', function(assert) {
+        const actionCount = sinon.spy();
+        const $list = $('#list').dxList({
+            items: [1, 2],
+            showSelectionControls: true,
+            selectByClick: false,
+            selectionMode: 'multiple',
+            onSelectionChanged: actionCount,
+        });
+
+        const $item1 = $list.find(`.${LIST_ITEM_CLASS}`).eq(0);
```

Comment:
We may find items = $list.find(`.${LIST_ITEM_CLASS}`) once and then trigger 'dxclick' for each of needed
$items.eq(0).trigger('dxclick')
$items.eq(q).trigger('dxclick')